syntax = "proto3";

service DemogRPC {
    rpc GetResponseFromPayload(PayloadMessage) returns (ResponseMessage) {}
}


message PayloadMessage {
    string customer_name = 1;
    uint32 id = 2;
    string created = 3;
    uint32 cost = 4;
    string id_uuid = 5;
}

message ResponseMessage {
    uint32 id = 1;
    int32 response_code = 2;
    string response_message = 3;
}